docs/src/content/docs/fr/components/using-components.mdx

```diff
@@ -0,0 +1,96 @@
+Ces balises ressemblent à des balises HTML, mais commencent par une lettre majuscule correspondant au nom utilisé dans votre instruction `import` :
+
+```mdx
+---
+# src/content/docs/exemple.mdx
+title: Bienvenue dans ma documentation
+---
+
+import { Icon } from '@astrojs/starlight/components';
+import CustomCard from '../../components/CustomCard.astro';
+
+<Icon name="open-book" />
+
+<CustomCard>
+	Les composants peuvent également contenir du **contenu imbriqué**.
+</CustomCard>
+```
+
+Starlight étant basé sur Astro, vous pouvez ajouter la prise en charge de composants construits avec n'importe quel [framework d'interface utilisateur pris en charge (React, Preact, Svelte, Vue, Solid, and Alpine)](https://docs.astro.build/fr/guides/framework-components/) dans vos fichiers MDX.
+Pour en savoir plus sur [l'utilisation de composants avec MDX](https://docs.astro.build/fr/guides/integrations-guide/mdx/#utilisation-de-composants-dans-mdx), consultez la documentation Astro.
+
+## Utilisation d'un composant avec Markdoc
+
+Ajoutez la prise en charge de la rédaction de contenu avec Markdoc en suivant notre [guide de configuration Markdoc](/fr/guides/authoring-content/#markdoc).
+
+En utilisant la préconfiguration Markdoc de Starlight, vous pouvez utiliser les composants intégrés à Starlight avec la syntaxe de balises `{% %}` de Markdoc.
+À la différence du format MDX, les composants dans les fichiers Markdoc n'ont pas besoin d'être importés.
+L'exemple suivant affiche le [composant de carte](/fr/components/cards/) de Starlight dans un fichier Markdoc :
+
+```markdoc
+---
+# src/content/docs/exemple.mdoc
+title: Bienvenue dans ma documentation
+---
+
+{% card title="Étoiles" icon="star" %}
+Sirius, Véga, Bételgeuse
+{% /card %}
+```
+
+Consultez la [documentation de l'intégration Astro Markdoc](https://docs.astro.build/fr/guides/integrations-guide/markdoc/#afficher-les-composants) pour plus d'informations sur l'utilisation des composants dans les fichiers Markdoc.
+
+## Composants intégrés
+
+Starlight fournit des composants intégrés par défaut pour des cas d'utilisation courants à une documentation.
+Ces composants sont disponibles dans le paquet `@astrojs/starlight/components` dans les fichiers MDX et dans la [préconfiguration Markdoc de Starlight](/fr/guides/authoring-content/#markdoc) dans les fichiers Markdoc.
+
+Utilisez la barre latérale pour obtenir une liste des composants disponibles et savoir comment les utiliser.
+
+## Compatibilité avec les styles de Starlight
+
+Starlight applique des styles par défaut à votre contenu Markdown, par exemple en ajoutant une marge entre les éléments.
+Si ces styles entrent en conflit avec l'apparence de votre composant, utilisez la classe `not-content` sur votre composant pour les désactiver.
+
+```astro 'class="not-content"'
+---
+// src/components/Exemple.astro
+---
+
+<div class="not-content">
+	<p>Contenu non affecté par les styles par défaut de Starlight.</p>
+</div>
+```
+
+## Props des composants
```

Comment:
To keep consistency with https://starlight.astro.build/fr/reference/overrides/#props-des-composants and Astro Docs

Reply:
“Props” is translated as ‘Propriétés’ in some files. Maybe change all the documentation _props_ -> _propriétés_ ?

Reply:
Oh, do you have specific links to this? We should definitely fix this. By quickly searching on the website, all "propriété(s)" are never used in the context of a component props.

<img width="666" alt="props" src="https://github.com/user-attachments/assets/b65177de-ab8b-4927-a25f-3e6ae8312c54">

--- 

This is indeed a tricky one. To give a bit more context on why we specifically picked "props" in this case, e.g. in the "Overrides Reference" page:

- This is a key concept in Astro and components.
- `props` is [heavily](https://docs.astro.build/fr/basics/astro-components/#props-de-composant) used in the French Astro documentation, same for `une props` or even `un prop`.
- If you're coming from the Astro documentation, you'll be familiar with this term for this feature and there is a high chance you'll be searching for it rather than "propriété".
- Other frameworks French documentation also tend to use `props` (e.g. [React](https://fr.react.dev/learn/passing-props-to-a-component)).

If we ever decide to change this, probably after a discussion with more people, I think we may want to do that on both the Starlight and Astro (which would be quite the effort) French documentation to keep consistency.

Reply:
> Oh, do you have specific links to this? We should definitely fix this. By quickly searching on the website, all "propriété(s)" are never used in the context of a component props.

This one for example _[La Propriété (props) page](https://docs.astro.build/fr/guides/routing/#la-propri%C3%A9t%C3%A9-props-page)_ or here _[transmettre des propriétés (Props)](https://docs.astro.build/fr/guides/framework-components/#transmettre-des-props-aux-composants-du-framework)_

Reply:
Oh, you mean in the Astro Docs, I thought you meant in the Starlight Docs.

Indeed, the Astro Docs is at the moment a mix of "props", "une props", "un prop", or "propriété" from what I can see. Could be worth discussing in the French Discord channel and see if it would be possible to agree on a fixed terminology and PR all instances. We did that at a few occasions in the Starlight repo, e.g. to rename all French example filenames, etc.

Reply:
We can leave props for the moment, and ask to French thread what is the best choice!